app/Website/src/Http/Controllers/ContactFormController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Website\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Website\Http\Requests\ContactFormRequest;
+use Knuckles\Scribe\Attributes\BodyParam;
+use Knuckles\Scribe\Attributes\Endpoint;
+use App\Website\Mail\ContactMail;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Http\JsonResponse;
+
+class ContactFormController extends Controller
+{
+    #[Endpoint(title: "Submit Contact Form", description: "This endpoint is used for submitting contact form data.")]
+    #[BodyParam(name: "first_name", type: "string", description: "The first name of the user.", required: true)]
+    #[BodyParam(name: "last_name", type: "string", description: "The last name of the user.", required: true)]
+    #[BodyParam(name: "email", type: "string", description: "The email address of the user.", required: true)]
+    #[BodyParam(name: "subject", type: "string", description: "The subject of the message.", required: true)]
+    #[BodyParam(name: "message", type: "string", description: "The body of the message.", required: true)]
+    #[BodyParam(name: "cf-turnstile-response", type: "string", description: "Cloudflare Turnstile ReCaptcha token")]
+    public function __invoke(ContactFormRequest $request): JsonResponse
+    {
+        $mailData = $request->validated();
+        $contactAddress = config('mail.contact_email');
+        Mail::to($contactAddress)->queue(new ContactMail($mailData));
```

Comment:
You are only using the variables once so you can inline that code and get rid of the variables.